conda/meta.yaml

```diff
@@ -1,19 +1,12 @@
 package:
   name: conda
-  version: 3.8.4
+  version:  '{{ environ["GIT_DESCRIBE_TAG"] }}.post{{ environ["GIT_DESCRIBE_NUMBER"] }}+{{ environ["GIT_DESCRIBE_HASH"] }}'
 
 source:
-  git_url: git@github.com:conda/conda.git
-  git_tag: 3.8.4
+  git_url: https://github.com/conda/conda.git
 
 build:
-  # These are present when the new environment is created
-  # so we have to exempt them from the list of initial files
-  # for conda-build to realize they should be included.
-  always_include_files:
-    - bin/activate
-    - bin/deactivate
-    - bin/conda
```

Comment:
Not on Windows I'm afraid... (or maybe not post 3.16?)

Tried with both `\` and `/` in case that was it, but neither works. 

```
Package: conda-3.18.3.post14+g74f2909-py27_0
source tree in: C:\dev\bin\Anaconda\conda-bld\work
Error: Glob bin\conda from always_include_files does not match any files
```

```
Package: conda-3.18.3.post14+g74f2909-py27_0
source tree in: C:\dev\bin\Anaconda\conda-bld\work
Error: Glob bin/conda from always_include_files does not match any files
```